### PR TITLE
pd-ctl: split operator support use key

### LINKED ISSUE #6601: pd-ctl: split operator should support user key
## Enhancement Task
In past, split operator command doesn't support user key: 
https://github.com/tikv/pd/blob/f8872fee1e7a9715865c6afc5e4ffc9ca5d6943d/tools/pd-ctl/pdctl/command/operator.go#L384-L387

### PR DESCRIPTION
**Pull request by bufferflies**
### What problem does this PR solve?

Issue Number: Close #6601

### What is changed and how does it work?

```commit-message
```

### Check List

Tests

- Unit test
- Integration test

Code changes

Side effects

Related changes

### Release note

```release-note
None.
```

### CODE REVIEW

File: tests/pdctl/operator/operator_test.go [hunk 1]
```diff
@@ -162,6 +162,13 @@ func TestOperator(t *testing.T) {
 			expect: "status: RUNNING",
 			reset:  []string{"-u", pdAddr, "operator", "remove", "3"},
 		},
+		{
+			// operator add split-region <region_id> [--policy=scan|approximate|usekey] [--key=xxx]
+			cmd:    []string{"-u", pdAddr, "operator", "add", "split-region", "3", "--policy=usekey", "--key=c"},
```
Comment (HuSharp) [inline tests/pdctl/operator/operator_test.go:167]:
```suggestion
			cmd:    []string{"-u", pdAddr, "operator", "add", "split-region", "3", "--policy=usekey", "--keys=c"},
```

Comment (bufferflies) [reply tests/pdctl/operator/operator_test.go]: fixed and add test

File: tests/pdctl/operator/operator_test.go [hunk 2]
```diff
@@ -162,6 +163,14 @@ func TestOperator(t *testing.T) {
 			expect: "status: RUNNING",
 			reset:  []string{"-u", pdAddr, "operator", "remove", "3"},
 		},
+		{
+			// operator add split-region <region_id> [--policy=scan|approximate|usekey] [--keys=xxx(xxx is hex encoded string)]
```
Comment (HuSharp) [inline tests/pdctl/operator/operator_test.go:167]: Perhaps all similar comments above need to be changed.

Comment (bufferflies) [reply tests/pdctl/operator/operator_test.go:167]: fixed

**Review by HuSharp — APPROVED** [review]
LGTM

File: tests/pdctl/operator/operator_test.go [hunk 3]
```diff
@@ -136,28 +136,28 @@ func TestOperator(t *testing.T) {
 			reset:  []string{"-u", pdAddr, "operator", "remove", "1"},
 		},
 		{
-			// operator add split-region <region_id> [--policy=scan|approximate]
+			// operator add split-region <region_id> [--policy=scan|approximate|usekey] [--keys=xxx(xxx is hex encoded string)]
```
[the remark this answers is not in the document]
Comment (bufferflies) [reply tests/pdctl/operator/operator_test.go:139]: only can use ：
var CheckPolicy_value = map[string]int32{
	"SCAN":        0,
	"APPROXIMATE": 1,
	"USEKEY":      2,
}

**Review by lhy1024 — APPROVED** [review]

File: tools/pd-ctl/pdctl/command/operator.go [hunk 4]
```diff
@@ -373,7 +374,7 @@ func splitRegionCommandFunc(cmd *cobra.Command, args []string) {
 
 	policy := cmd.Flags().Lookup("policy").Value.String()
 	switch policy {
-	case "scan", "approximate":
+	case "scan", "approximate", "usekey":
```
Comment (nolouch) [inline tools/pd-ctl/pdctl/command/operator.go:377]: user key?

Comment (bufferflies) [reply tools/pd-ctl/pdctl/command/operator.go:377]: https://github.com/tikv/pd/blob/f8ca1e83f825e29b5202c392ae120645286a01d8/server/handler.go#L803

**Pull request merged**